docs/user/reconciliation.md

```diff
@@ -93,22 +93,22 @@ So the graph nodes are all parallelized with the by-number ordering flattened ou
 
 For the usual reconciliation loop (neither an upgrade between releases nor a fresh install), the flattened graph is also randomly permuted to avoid hanging on ordering bugs.
 
-## Synchronizing the graph
+## Reconciling the graph
```

Comment:
```suggestion
## Reconciling the cluster version
```
nit: "graph" sounds a bit ambiguous to me  - I've immediately though of "updates graph" instead "manifests graph"

Reply:
This is a backport PR.  Drop an `/lgtm`, and then PR master with your change to "manifest graph", which sounds good to me.